README.md

```diff
@@ -27,7 +39,7 @@ To generate a configuation file template, run the command below in your terminal
 cargo run -- print-config
```

Comment:
Since we moved the code to a workspace, `cargo` needs to know what binary to execute when passing it commands. This would now look like `cargo run --bin steward print-config`. This is also true of the other `cargo` commands in the readme.

Reply:
done!